ethereum/contracts/IncentivizedInboundChannel.sol

```diff
@@ -74,8 +91,15 @@ contract IncentivizedInboundChannel is InboundChannel {
             (bool success, ) =
                 _messages[i].target.call{value: 0, gas: MAX_GAS_PER_MESSAGE}(_messages[i].payload);
 
+            _rewardAmount = _rewardAmount + _messages[i].fee;
```

Comment:
so, the reward is based on adding up all message fees. it would be worth double checking/auditing exactly how this fee variable can be controlled/changed.

so far it looks like it can only be controlled/changed by submitting a tx to the parachain to send a message through the channel, and only when burning the equivalent amount of eth on the parachain side equal to the fee that has been preset in config. looks good, though this is a critical field/code path, so plz audit yourself again too @philipstanislaus @vgeddes cc @Rizziepit

Reply:
The default per-message fee is set on the parachain side via genesis storage for the incentivized outbound channel. This fee can be updated though via the `set_fee` extrinsic (which is locked down via governance).